toolkit/sidechain/domain/src/lib.rs

```diff
@@ -338,7 +446,7 @@ impl MainchainKeyHash {
 /// EDDSA signature, 64 bytes.
 pub const MAINCHAIN_SIGNATURE_LEN: usize = 64;
 
-#[derive(Clone, TypeInfo, PartialEq, Eq, Hash)]
+#[derive(Clone, DecodeWithMemTracking, TypeInfo, PartialEq, Eq, Hash)]
```

Comment:
here should be fine to derive it, even if we have custom `Decode` impl, but I will double check once again